extensions/simple-mcp-server/requirements.txt

```diff
@@ -0,0 +1,42 @@
+# This file was autogenerated by uv via the following command:
+#    uv export -o requirements.txt --no-hashes
```

Comment:
OMG, I didn't know that uv could export directly to a requirementx.txt like this. You just saved future me so much pain. :1st_place_medal: